integration/src/main/java/scripts/utils/ClientTestUtils.java

```diff
@@ -226,6 +229,10 @@ private static ApiClient buildClient(
       apiClient.setAccessToken(accessToken.getTokenValue());
     }
 
+    ClientConfig clientConfig = new ClientConfig();
```

Comment:
Can you comment what all this is for? Why do we only need to do it in the test config? I'd have thought you'd want the same config for test and production.

Reply:
Good question. Let me check. I think this snippet may need to go into a custom modification of the generated code. Otherwise, every client would have to do this I think.